src/lib/config.js

```diff
@@ -32,9 +47,66 @@ async function config() {
       console.error(err);
     }
     log(chalk.red(str));
-    return;
+    process.exit(1);
   }
+}
 
+async function createLightnetDeployAlias(projectRoot, deployAliasesConfig) {
+  const networkId = 'testnet';
+  const deployAliasPrefix = 'lightnet';
+  const minaGraphQlEndpoint = 'http://localhost:8080/graphql';
+  let nextAliasNumber = 1;
+  await step(`Check Mina GraphQL endpoint availability`, async () => {
+    const minaGraphQlEndpointIsAvailable =
+      await isMinaGraphQlEndpointAvailable(minaGraphQlEndpoint);
+    if (!minaGraphQlEndpointIsAvailable) {
+      throw new Error(
+        `Mina GraphQL endpoint ${minaGraphQlEndpoint} is not available.`
+      );
+    }
+  });
+  while (
+    deployAliasesConfig.deployAliases[`${deployAliasPrefix}${nextAliasNumber}`]
+  ) {
+    nextAliasNumber++;
+  }
+  const deployAliasName = `${deployAliasPrefix}${nextAliasNumber}`;
+  const feePayerPath = `${Constants.feePayerCacheDir}/${deployAliasName}.json`;
+  if (!fs.existsSync(feePayerPath)) {
+    await step(
+      `Create zkApp fee payer key pair at ${feePayerPath}`,
+      async () => {
+        const minaNetworkInstance = Mina.Network({
+          networkId,
+          mina: minaGraphQlEndpoint,
+          lightnetAccountManager: 'http://localhost:8181',
```

Comment:
this is a nit but could we move the endpoints for lightnet in a constants file or something similar? I think it'd make it easier to understand that these are expected defaults

Reply:
Will extract it into constants, thanks.

Reply:
+1 please.

Reply:
Done.